01_P/P_2_1_2_04/sketch.js

```diff
@@ -18,40 +18,37 @@
 
 /**
  * moving corners of rectangles in a grid
- *   
+ *
  * MOUSE
  * position x          : corner position offset x
  * position y          : corner position offset y
  * left click          : random position
- * 
+ *
  * KEYS
  * s                   : save png
  */
- 
+'use strict';
 
 var tileCount = 20;
 var rectSize = 30;
 
 var actRandomSeed = 0;
 
-
 function setup(){
   createCanvas(600, 600);
+  colorMode(HSB, 360, 100, 100, 100);
```

Comment:
Moved `colorMode`, `noStroke` and `fill` into `setup` because it only has to be set once.